src/commons/utils/jira_utils.py

```diff
@@ -34,9 +34,7 @@
 from requests.exceptions import RequestException
 from requests.packages.urllib3.util.retry import Retry # pylint: disable=import-error
 
-from src.commons.constants import ROOT
-
-LOGGER = logging.getLogger(ROOT)
+LOGGER = logging.getLogger(__name__)
```

Comment:
Revert this change as we are logging common & utils related logs in CORIO log file.